aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -72,6 +72,30 @@ define([],function() {
       CLEAR_SELECTED_ITEMS: "ALF_CLEAR_SELECTED_ITEMS",
 
       /**
+       * This can be published to make a request to create a dialog. It is typically handled by the
+       * [DialogService]{@link module:alfresco/services/DialogService}.
+       *
+       * @instance
+       * @type {string}
```

Comment:
Yes, I think you're right... but we've missed this for all the others, we probably need a separate issue to go over all the others that we've missed. I'll update the current one as an example for future reference.